solr/solr-ref-guide/src/basic-authentication-plugin.adoc

```diff
@@ -223,6 +223,16 @@ Http2SolrClient client = new Http2SolrClient.Builder(solrUrl)
 QueryResponse rsp = req.process(client);
 ----
 
+CloudHttp2SolrClient supports receiving an `Http2SolrClient.Builder` instance for creating it's internal client, so to set the credentials at the client level you could use a code like:
```

Comment:
"it's" should be "its" instead, but otherwise go ahead with this when you're ready IMO.